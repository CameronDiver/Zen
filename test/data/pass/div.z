let a = 3.0/3.0;
printf("%.3f\n", a);
printf("%i\n", 3/2);
printf("%i\n", 10/2);
printf("%.3f\n", 10/2.0);
printf("%.3f\n", 10.0/2);
