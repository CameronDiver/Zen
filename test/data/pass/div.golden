1.000
1
5
5.000
5.000
